csharp-advanced-topics/ActionFuncDelegates/ActionAndFuncDelegates/ActionAndFuncDelegates.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
```

Comment:
We still haven't added this to the docs, but we try to use the latest framework version without exception. Once you start working on the real articles, please use .NET 6.